data/tribes/initialization/amazons/units.lua

```diff
@@ -1380,8 +1407,12 @@ wl.Descriptions():new_tribe {
             no_scouting_building_connected = pgettext("amazons_building", "You need to connect this flag to a scout’s hut before you can send a scout here."),
             -- TRANSLATORS: Purpose helptext for an Amazon production site: Scout's Hut
             purpose = pgettext("amazons_building", "Explores unknown territory."),
-            -- TRANSLATORS: Performance helptext for an Amazon production site: Scout's Hut
-            performance = pgettext("amazons_building", "The scout pauses %s before going to work again."):bformat(format_seconds(30))
+            performance = {
+               -- TRANSLATORS: Performance helptext for an Amazon production site: Scout's Hut, part 1
+               pgettext("amazons_building", "The scout makes exploration between %1$s and %2$s."):bformat(format_minutes_seconds(1, 52), format_minutes_seconds(4, 5)),
```

Comment:
"consumes a ration"?

Reply:
Should've posted this here: OK, how about "One exploration trip of the scout takes between…"?